bgpd/bgp_evpn_vty.c

```diff
@@ -1006,7 +1006,7 @@ static int bgp_show_ethernet_vpn(struct vty *vty, struct prefix_rd *prd,
 	json_object *json_scode = NULL;
 	json_object *json_ocode = NULL;
 
-	bgp = bgp_get_default();
+	bgp = bgp_get_evpn();
```

Comment:
The one problem i have with all the show commands is that we are showing info from a non-default vrf even though a "vrf" has not been provided in the CLI. I don't have an immediate alternate suggestion to this but curious about @vivek-cumulus and other experts' opinion.

Reply:
I do agree, as this breaks the coherency with the rest of the commands. However, we still support a single EVPN VRF, and all commands bear the `l2vpn evpn` mark in them.

The main alternative is duplicating all the commands in separate handlers to take a VRF as an argument, which would be used for at most a single VRF.

We need your input guys!

Reply:
+1 @AnuradhaKaruppiah 

I think taking VRF as an optional input parameter for these commands may be a better alternative. Not sure if this is best though. @vivek-cumulus or @donaldsharp

Reply:
srimohans,  +1 for your proposal.
adding VRF to show evpn commands is good.